### PR TITLE
More logging with sqlplus

### PR DESCRIPTION
**Pull request by gip**

### CODE REVIEW

File: src/lib/db.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ function executeSQLPlus (sqlFilepath) {
     const host = connection.host
     const port = connection.port
     const database = connection.database
-    const login = user + '/' + password + '@' + host + ':' + port
+    // TODO: Oracle connection string passed as configuration to ledger
+    const login = user + '/' + password + '@' + host + '/xe:' + port
```
Comment (clark800) [inline src/lib/db.js]: This conflicts with the documentation: https://docs.oracle.com/cd/B19306_01/server.102/b14357/qstart.htm#i1055567

Comment (gip) [inline src/lib/db.js]: Yeah, but the amazing thing is that actually solved the connection issue. I found another way to do that so this change will be undone in next commit.

File: src/lib/db.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ function executeSQLPlus (sqlFilepath) {
     const host = connection.host
     const port = connection.port
     const database = connection.database
+    // TODO? Oracle connection string passed as configuration to ledger
```
Comment (clark800) [inline src/lib/db.js]: This is already the way it is, isn't it?

File: src/lib/db.js [hunk 3]
```diff
@@ -35,6 +36,8 @@ function executeSQLPlus (sqlFilepath) {
     const command = '/opt/oracle/instantclient/sqlplus'
     const args = [url, '@' + sqlFilepath]
     const process = spawn(command, args, {env})
+    process.stdout.on('data', (data) => console.log(data.toString()))
```
Comment (clark800) [inline src/lib/db.js]: I don't know if we always want to show all this; could we send it to the debug logger so it is only shown in debug mode?

**Comment by MatthewPhinney** [comment]
What's the status of this PR?

**Pull request closed without merging**